scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua

```diff
@@ -1,57 +1,131 @@
------------------------------------
---  Area: Ru'Lude Gardens
---  NPC:  Nomad Moogle
---  Type: Adventurer's Assistant
---  @pos 10.012 1.453 121.883 243
 -----------------------------------
-package.loaded["scripts/zones/RuLude_Gardens/TextIDs"] = nil;
```

Comment:
You originally added a nil for settings.lua which wasn't needed. now you are removing TextIDs.

You misunderstand: 
TextIDs.lua needed that line. The settings did not.
When you nil it, that means it is unloaded from memory and when you require it, it is reloaded.
TextIDs.lua is special and has to be done this way, the others are not.